src/tasks/daily/misc/daily_logistics_mixin.py

```
@@ -52,20 +52,16 @@ def claim_mail(self):
-
-        area = areas_list[0]
-        self.to_model_area(area, "仓储节点")
-
         if not self.wait_click_ocr(
                 match=self.lang.daily_routine_mixin.k_41a9fd98,
                 box=self.box.top_left,
                 time_out=5
         ):
-            self.log_info(f"'未找到我转交的委托'节点，返回主界面")
-            self.ensure_main()
+            self.log_info("未找到‘我转交的委托’节点，跳过领取")
```

Ruff (0.16.0)

[warning] String contains ambiguous `‘` (LEFT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)